src/Controller/CkeditorAdminController.php

```diff
@@ -28,14 +29,20 @@
 /**
  * @phpstan-extends CRUDController<object>
  */
-final class CkeditorAdminController extends CRUDController
-{
+final class CkeditorAdminController extends CRUDController{
+
+    public function __construct(ContainerInterface $container)
```

Comment:
There is something wrong here, to me the purpose of the ` getSubscribedServices` is to not use the container.

Reply:
>There is something wrong here, to me the purpose of the getSubscribedServices is to not use the container.

Yo may be right that the constructor here is not necessary, let me check.
However please note the type of container. 
use Psr\Container\ContainerInterface

[Defining a Service Subscriber]
**DOCS**
(https://symfony.com/doc/current/service_container/service_subscribers_locators.html#defining-a-service-subscriber)
_First, turn CommandBus into an implementation of [ServiceSubscriberInterface](https://github.com/symfony/symfony/blob/6.1/src/Symfony/Contracts/Service/ServiceSubscriberInterface.php). Use its getSubscribedServices() method to include as many services as needed in the service subscriber and change **the type hint of the container to a PSR-11 ContainerInterface:**_

The service subscriber pattern you use here relies on PSR container which is a thinner version of the container.
It's exactly the reason why we need to use container as a service alias not class name in the router - to let the container load services required by 
public static function getSubscribedServices(): array
before calling action.
At least my debug sessions let me to this conclusion and the working version.

Reply:
If the constructor was needed, it would already be needed in SonataAdmin here https://github.com/sonata-project/SonataAdminBundle/blob/4.x/src/Controller/CRUDController.php

Reply:
yep, you are right, sorry that is leftover from first attemtps to solve missing conatiner error.
We dont need it because we call setContainer on CRUD conroller
            <call method="setContainer">
                <argument type="service" id="Psr\Container\ContainerInterface"/>
            </call>